postgres/postgresql.conf.erb

```diff
@@ -33,13 +32,20 @@ lc_numeric = 'en_AU.UTF-8'
 lc_time = 'en_AU.UTF-8'
 default_text_search_config = 'pg_catalog.english'
 
+idle_in_transaction_session_timeout = '120s'
+
 # These settings only apply to the master.
-wal_level = hot_standby
+wal_level = replica
```

Comment:
Slightly larger WAL files don't sound that bad, let's set the `wal_level` to `logical` so we have the option to experiment with logical replication if we want to.